nav2_experimental/nav2_rl/nav2_turtlebot3_rl/gazebo_interface.py

```diff
@@ -32,6 +32,7 @@ class GazeboInterface(Env):
     def __init__(self):
         super()
         self.GazeboInterface = GazeboInterface
+        rclpy.init()
```

Comment:
This can only be done once in the program. Is this really the right place to do this? It seems it should be in main() instead, or at least some class that is higher in the stack.

Reply:
Yes.  This needs to be removed.  It's already being done in the navigator_ddog which has the main.